pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -91,14 +91,14 @@ func NewPodNetworkAvalibilityInvariant(info monitortestframework.MonitorTestInit
 func (pna *podNetworkAvalibility) PrepareCollection(ctx context.Context, adminRESTConfig *rest.Config, recorder monitorapi.RecorderWriter) error {
 	deploymentID := uuid.New().String()
 
-	openshiftTestsImagePullSpec, err := GetOpenshiftTestsImagePullSpec(ctx, adminRESTConfig, pna.payloadImagePullSpec, nil)
```

Comment:
This doesn't seem to do anything significant?

Reply:
oc is passed into the call on the next line for GetOpenshiftTestsImagePullSpec.  That had to be reworked as initializing oc in DetermineRegistryAuthFilePath led to ginkgo panics

https://github.com/openshift/origin/pull/30007/commits/0e992adb8f43ebb9d925e64c719aaa62a9e9f801

Reply:
ooh, I can't believe I missed it